packages/grafana-ui/src/components/Forms/RadioButtonList/RadioButtonDot.tsx

```diff
@@ -5,17 +5,27 @@ import { GrafanaTheme2 } from '@grafana/data';
 
 import { useStyles2 } from '../../../themes';
 
-export interface RadioButtonDotProps {
+export interface RadioButtonDotProps<T> {
   id: string;
   name: string;
   checked?: boolean;
+  value?: T;
   disabled?: boolean;
   label: React.ReactNode;
   description?: string;
   onChange?: (id: string) => void;
 }
 
-export const RadioButtonDot = ({ id, name, label, checked, disabled, description, onChange }: RadioButtonDotProps) => {
+export const RadioButtonDot = <T extends string | number | readonly string[]>({
```

Comment:
Same question regarding the default type, would a default type prevent this from being a breaking change?

Reply:
It doesn't seem to be a breaking change. If you don't specify the type, it defaults to `uknown` or infer the type from the `value` prop if passed to the component

Reply:
Ok ok, so if you don't pass any value it won't complain? Good! Thanks 🙏🏻

Reply:
Exactly. It doesn't complain if you don't pass any value